src/view/index.js

```diff
@@ -1,2 +1,38 @@
+/**
+ * WordPress dependencies
+ */
+import domReady from '@wordpress/dom-ready';
+
+/**
+ * Internal dependencies
+ */
 import './style.scss';
 import './patterns.scss';
+
+const manageForms = container => {
+	const forms = Array.from( container.querySelectorAll( 'form.popup-action-form' ) );
+	forms.forEach( form => {
+		form.addEventListener( 'submit', event => {
+			const XHR = new XMLHttpRequest();
+			const inputs = Array.from( form.querySelectorAll( 'input' ) );
+			const pairs = inputs.map(
+				( { name, value } ) => encodeURIComponent( name ) + '=' + encodeURIComponent( value )
+			);
+			const data = pairs.join( '&' ).replace( /%20/g, '+' );
```

Comment:
It'd be safer to create the data string using `@wordpress/url`'s [`addQueryArgs`](https://github.com/WordPress/gutenberg/blob/master/packages/url/README.md#addQueryArgs):
```js
const query = inputs.reduce( ( acc, { name, value } ) => {
	acc[ name ] = value;
	return acc;
}, {} );
const data = addQueryArgs( '', query ).replace( /^\?/, '' );
```

Reply:
> It'd be safer to create the data string using `@wordpress/url`'s [`addQueryArgs`](https://github.com/WordPress/gutenberg/blob/master/packages/url/README.md#addQueryArgs):

Tried this and I get this error, any idea why?

```
Uncaught TypeError: Cannot read property 'addQueryArgs' of undefined
    at HTMLFormElement.eval (index.js:34)
```

Reply:
That's super strange, I have no idea why it happens but it does also on my machine. I've checked the `node_modules/@wordpress/url` folder and it looks fine… 
Anyway, in this case using [`qs`](https://www.npmjs.com/package/qs) instead of `addQueryArgs ` would be more readable:
```js
import { stringify } from 'qs';
// …
const data = stringify( query );
```

Reply:
> Anyway, in this case using [`qs`](https://www.npmjs.com/package/qs) instead of `addQueryArgs ` would be more readable:

This works, but `qs-stringify` adds about 10k. Since this is for frontend use the file should be as lean as possible. What's your preferred approach without requiring a library?